### ISSUE TITLE
Gramps Web Sync failure on Login tab feedback

### ISSUE THREAD

**Issue by emyoulation** [DavidMStraub/gramps-web-sync#23]
The Login tab needs feedback of whether the GrampsWeb server specified is valid. And separately whether the Account and Password were valid. (GrampsWeb on the serverside should only do a single "not valid" error response of Account and Password test so that it is not possible to brute force test for valid Accounts, then look for Passwords on that account.)   

Note that there is a Feature Request for the new 5.2 Addon Manager to give validation feedback on AddonList server addresses.  Any feature for Gramps Web Sync should follow the lead of how that is implemented.
 
Suggest that you "follow" that MantisBT bug report and add a note linking back to this GitHub issue.
https://gramps-project.org/bugs/view.php?id=12974

Note that a failed Login step (in my case, the laptop was not connected to a network) jumps straight to a BLANK Summary tab. No information is an insufficient "Summary".

**Comment by DavidMStraub** [comment]
I would be so happy if somebody could support me on this. I am a relly poor Gtk developer.

**Comment by emyoulation** [comment]
Expanding the docs should be a step toward recruiting someone to do some work on it. If we can see what is expected, we can see where to improve the UI and contribute towards that.

I wanted to get a screen capture of each stage.  But only 3 stages are accessible without an Account ... and one of those was not representative. (The blank summary.) 

My eventual plan is to acquire a Raspberry Pi for experimenting

**Comment by hdholm** [comment]
Attempted to deal with this in https://github.com/gramps-project/addons-source/pull/749  Note that the changes to GrampsWebSync do nothing about attempted brute force, because really that has to be handled on the API server side.  Anyone could build their own Sync tool without checks.

**Comment by DavidMStraub** [comment]
> Note that the changes to GrampsWebSync do nothing about attempted brute force, because really that has to be handled on the API server side.  Anyone could build their own Sync tool without checks.

That's correct, and just to be clear (since it wasn't clear to me from your statement whether you're implying there is a problem) it **is handled** on the API server side:

https://github.com/gramps-project/gramps-web-api/blob/master/gramps_webapi/api/resources/token.py#L73

**Comment by hdholm** [comment]
Sorry, I never meant to imply that the server was doing anything wrong, just that the PR doesn't address that.  It was really trying to respond to (my misreading of) comments in the original issue description.

**Comment by hdholm** [comment]
I think this is closed by https://github.com/gramps-project/addons-source/pull/749
